src/main/riddl/Venues/venueTypeDefinitions.riddl

```diff
@@ -0,0 +1,37 @@
+type Status is any of {Draft, Open, Closed} briefly "Internal status of a Venue."
+
+type Info is {
+	orgId: OrganizationId briefly "Venues are defined by BaseOrgs. venueId and orgId make a composite key",
+  name: String,
+  address: Address,
+  isPrivate: Boolean briefly "Is this location accessible outside of the organization owning its venue?"
+}
+
+type UpdateInfo is {
+	orgId: OrganizationId?,
+  name: String?,
+  address: Address?
+}
+
+type VenueReservationsSchedule is {
```

Comment:
I don't think the idea of a reservation schedule belongs here. I think there is a projection that would provide this info.